frontends/bsd/src/app_root.tsx

```diff
@@ -640,6 +640,10 @@ export function AppRoot({ card, hardware }: AppRootProps): JSX.Element {
               hasBatches={status.batches.length > 0}
               isTestMode={isTestMode}
               toggleTestMode={toggleTestMode}
+              hasExportedBackupForAllBatches={status.batches.every(
+                (b) => b.exportedAt
+              )}
+              exportedAt={status.batches[0] ? status.batches[0].exportedAt : ''}
```

Comment:
Could be abbreviated:
```suggestion
              exportedAt={status.batches[0]?.exportedAt}
```
No need for the `''` as `undefined` should suffice.